ehrql/debugger.py

```diff
@@ -156,5 +156,11 @@ def related_event_columns_to_records(columns):
         for row_id in row.keys():
             record = {"patient_id": patient_id, "row_id": row_id}
             for i, column in enumerate(columns, start=1):
-                record[f"series_{i}"] = column[patient_id][row_id]
+                record[f"series_{i}"] = render_value(column[patient_id][row_id])
             yield record
+
+
+def render_value(value):
+    if value is None:
+        value = ""
+    return value
```

Comment:
Rather than reimplement this, it would be good to import the version from `in_memory_database.py`.